### ISSUE TITLE
Data normalization for pretrained models

### ISSUE THREAD

**Issue by clkruse** [zhu-xlab/SSL4EO-S12#2]
I'm interested in testing your pretrained models on Sentinel data that I have extracted. Is there a pre-defined normalization scheme for these models?

**Comment by calebrob6** [comment]
Hey @clkruse, here's a notebook in which I use torchgeo with these models to do linear probing on eurosat -- https://gist.github.com/calebrob6/44f2e42017e2d192e837f0a1cd526c50. I found / 10,000 is likely what you want!

**Comment by clkruse** [comment]
Thanks fellow Caleb (@calebrob6)! This notebook seems perfect

**Comment by calebrob6** [comment]
Glad to help Caleb (@clkruse) -- let me know if you have any questions!

**Comment by wangyi111** [comment]
Thanks @calebrob6 for helping with the answer!

@clkruse We did use a simple division of 10000 for the pre-trained models. In fact, we found the influence of data normalization depends on the downstream dataset, but not much. You can check the appendix C 1) data preprocessing in the [pre-print paper](https://arxiv.org/abs/2211.07044) for details.
